lib/components/chat_history/twitch/message.dart

```diff
@@ -99,7 +110,13 @@ class TwitchMessageWidget extends StatelessWidget {
         ),
       );
     } else if (token is UserMentionToken) {
-      yield TextSpan(text: "@${token.username}", style: tagStyle);
+      final userModel = Provider.of<UserModel>(context, listen: false);
+      final loginChannel = userModel.userChannel!.displayName;
+      if (token.username == loginChannel) {
```

Comment:
`if (token.username.toLowerCase() == loginChannel.toLowerCase())`, we'll want to compare these case insensitive.